### ISSUE TITLE
Word wrapping around images doesn't work when building latexpdf output.

### ISSUE THREAD

**Issue by DavidGriffith** [i6/ibg#8]
At least with the drop capitals, word wrapping around images doesn't work when building latexpdf output.  This works fine for html output.  Instead of wrapping, the system just leaves the image above the paragraph to be wrapped.

**Comment by zondo** [comment]
This might be better handled in LaTeX using the [actual font](http://www.tug.dk/FontCatalogue/apicturealfabet/).

**Comment by DavidGriffith** [comment]
Indeed!  In Sphinx, how does one specify to do something one way for HTML and another way for LaTeX?

**Comment by zondo** [comment]
There are several ways:
- You can include content or not based on [tags](http://www.sphinx-doc.org/en/stable/markup/misc.html#including-content-based-on-tags) (either that you define yourself, or that come built-in, such as the builder name: html, latex, etc)
- There's an extension called [ifconfig](http://www.sphinx-doc.org/en/stable/ext/ifconfig.html) which can include content or not depending on settings in `conf.py`
- You can also send raw output to a particular builder at a particular point via the `raw` directive.  For example:
  
  ```
  .. raw:: latex
  
     \mylatexcommand
  ```

There's some examples in the latest version of `index.rst`.

**Comment by zondo** [comment]
Problem with using the emerald font is that it's non-standard, and I can't find a debian package that contains it.  (Not really surprising, since the license is non-free.)  So people would have to manually download and set up the font themselves to use it. :-(  This might be OK if LaTeX could be made to use a fallback option if it's not installed, instead of barfing.

**Comment by DavidGriffith** [comment]
Does the Emerald City Fontwerks even exist anymore?  I see a webpage with little more than an email address at http://users.speakeasy.net/~ecf/.

**Comment by DavidGriffith** [comment]
I'll go through these two resources and hopefully come up with a relatively painless way to install the Emerald package:

http://tex.stackexchange.com/questions/248464/installing-emerald-fonts-texlive-wont-find-font-files
http://newtips.co/tex/questions/110797/how-to-install-font-from-the-font-catalogue-in-debian.html

**Comment by DavidGriffith** [comment]
That was easier than I thought it would be.  This script will do the trick:

```
#!/bin/sh

if [ `ps -o uid= $$` -ne 0 ] ; then
        echo "This script must be run as root"
        exit 1
fi

TEXMFLOCAL=`kpsewhich -var TEXMFLOCAL`

echo "Installing to $TEXMFLOCAL..."

install -d $TEXMFLOCAL $TEXMFLOCAL/web2c

cp -Rp fonts $TEXMFLOCAL
cp -Rp other $TEXMFLOCAL
cp -Rp tex $TEXMFLOCAL

echo "Map emerald.map" >> $TEXMFLOCAL/web2c/updmap.cfg

mktexlsr $TEXMFLOCAL

updmap-sys
```

Or should I come up with a .deb package instead?

**Comment by DavidGriffith** [comment]
Fixed in 00e42bd8b271a01dd1a4a7c6b90c12bf5b52ae76

**Comment by zondo** [comment]
Wow, that was very quick!  I'll give it all a try when I get a spare moment.
